docs/copied-from-beats/shared-kibana-config.asciidoc

```diff
@@ -109,8 +109,7 @@ Example configuration:
 
 [source,yaml]
 ----
-setup.kibana.host: "192.0.2.255:5601"
-setup.kibana.protocol: "https"
+setup.kibana.host: "https://192.0.2.255:5601"
```

Comment:
I am confused now with `http` and `https` usage. I know you haven't written that, but why did that change and why do we have a mix? I would not pull this in.